torch/nn/modules/container.py

```diff
@@ -62,6 +62,11 @@ def __getitem__(self, idx):
     def __len__(self):
         return len(self._modules)
 
+    def __dir__(self):
+        keys = super(Sequential, self).__dir__()
+        keys = (key for key in keys if not key.isdigit())
+        return sorted(keys)
```

Comment:
I don't think we need to sort again. `keys` is returned by `__dir__` so it's guaranteed to be sorted, and you're only filtering out certain entries

Reply:
You're right, I've fixed it.